.changelog

```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Reference Guide: removed underscores in page titles and nav bar links.
```

Comment:
Did users see such underscores? If not, the changelog entry is not needed, as it's not relevant for them. Otherwise it can stay.